app/views/browse/deployment-config.html

```diff
@@ -84,7 +84,7 @@
                     </li>
                   </ul>
                 </div>
-                <!-- <span ng-if="deploymentConfig.status.details.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-trigger="hover" dynamic-content="{{deploymentConfig.status.details.message}}"></span> -->
+                <span ng-if="deploymentConfigMissing" class="pficon pficon-warning-triangle-o" data-toggle="tooltip" aria-hidden="true" data-original-title="This deployment configuration no longer exists"></span>
```

Comment:
Are we not already showing a message somewhere that the DC does not exist?  I don't think I would treat this the same way as the deployment having a missing deployment config.  The fact that the page you are viewing is for an object which no longer exists is a bigger issue.